src/app/core/models/db/accounting-export.model.ts

```
@@ -250,32 +251,32 @@ export class AccountingExportModel {
+  static constructNetsuiteExportUrlAndType(expenseGroup: ExpenseGroup, translocoService: TranslocoService): [string, string] {
     const words: string[] = expenseGroup.response_logs?.type.split(/(?=[A-Z])/);
-    const exportType = new SentenceCasePipe().transform(words?.join(' '));
+    const exportType = new SentenceCasePipe(translocoService).transform(words?.join(' '));
 
     return [expenseGroup.export_url, exportType];
   }
 
-  static constructExportUrlAndType(appName: AppName, expenseGroup: ExpenseGroup): [string, string] {
+  static constructExportUrlAndType(appName: AppName, expenseGroup: ExpenseGroup, translocoService: TranslocoService): [string, string] {
     if (appName === AppName.QBO) {
       return this.constructQBOExportUrlAndType(expenseGroup);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)